internal/pkg/v2/dto/core/data/event.go

```diff
@@ -0,0 +1,41 @@
+package data
+
+import (
+	dtoBase  "github.com/edgexfoundry/edgex-go/internal/pkg/v2/dto/common/base"
+	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
+)
+
+// AddEventRequest defines the Request Content for POST event DTO. This object and its properties correspond to the
+// AddEventRequest object in the APIv2 specification.
+type AddEventRequest struct {
+	dtoBase.Request `json:",inline"`
+	Event           contract.Event
```

Comment:
Should Event be inline according to the API document?

Reply:
Updated.